javalib/src/main/scala/java/util/Random.scala

```diff
@@ -13,10 +13,12 @@ class Random(seed_in: Long) extends AnyRef with java.io.Serializable {
   private var nextNextGaussian: Double = _
   private var haveNextNextGaussian: Boolean = false
 
-  setSeed(seed_in)
+  _setSeed(seed_in)
```

Comment:
You cannot change this. This alters the behavior of `java.util.Random`.

Instead, you have to use a flag `initialized` that is set to `true` in the constructor, and let `ThreadLocalRandom.setSeed()` be called if `!initialized` (and call `super`). And throw `UnsupportedOperationException` only if `initialized`.